core/src/sumeragi/message.rs

```diff
@@ -89,14 +83,12 @@ pub enum Message {
 #[non_exhaustive]
 pub struct BlockCreated {
     /// The corresponding block.
-    pub block: VersionedCandidateBlock,
+    pub block: PendingBlock,
```

Comment:
Why not `VersionedPendingBlock`?

Reply:
Blocks by consensus are always of the "most recent" version. And instead of promoting all versions to the latest version we should just use the "latest version" in the code. We can make an informed decision about how to handle live upgrades to a network in the future. The current approach gave the impression that this scheme was thought out when I don't think it was. Having a versioned committed block is important though.

I forgot to remove VersionedPendingBlock and have now done so.